src/poset/event.proto

```diff
@@ -7,6 +7,13 @@ enum TransactionType {
   PEER_REMOVE = 1;
 }
 
+message Event {
```

Comment:
Are `Event` ever serialized? It is been a while since I last worked in this repo, but no as far as I can recall, does that still hold?

Reply:
But this is most likely a mistake that needs to be fixed in the future. Event does not need to be serialized.
```
func (e *Event) ProtoMarshal() ([]byte, error) {
	var bf proto.Buffer
	bf.SetDeterministic(true)
	if err := bf.Marshal(e); err != nil {
		return nil, err
	}
	return bf.Bytes(), nil
}

func (e *Event) ProtoUnmarshal(data []byte) error {
	return proto.Unmarshal(data, e)
}
```